components/automate-ui/src/app/helpers/event-feed/event-helper.ts

```diff
@@ -0,0 +1,88 @@
+
+import { endsWith, replace } from 'lodash/fp';
```

Comment:
This is a refactor to have the logic of getting the correct icon for an event type and event type display name in one place. This is also adding the new event type of "local_user".